cycledash/static/js/examine/components/VCFTable.js

```diff
@@ -471,17 +480,28 @@ var VCFComment = React.createClass({
 var VCFCommentHeader = React.createClass({
   propTypes: {
     record: React.PropTypes.object.isRequired,
+    igvLink: React.PropTypes.string,
     handleOpenViewer: React.PropTypes.func.isRequired,
     handleEdit: React.PropTypes.func.isRequired,
     handleDelete: React.PropTypes.func.isRequired
   },
   render: function() {
+    var r = this.props.record,
+        locusParam = `&locus=${r.contig}:${r.position}`,
+        loadIGVLink = this.props.igvLink + locusParam,
+        jumpLink = loadIGVLink.replace(/\/load.*/, '/goto?') + locusParam;
+
     return (
       <div className='comment-header'>
         <a className='dalliance-open'
-           onClick={() => {this.props.handleOpenViewer(this.props.record);}}>
+           onClick={() => {this.props.handleOpenViewer(r);}}>
           Open Pileup Viewer
         </a>
+        <span className='igv-links'>
+          IGV: <a href={loadIGVLink}>Load at Locus</a>&nbsp;
```

Comment:
No need to do this in this PR unless you'd like + it's trivial, but could be nice to disable the "load at locus" once it's been clicked once, as after the first time you load, you'll just need the "jump" button.